### DISCUSSION TITLE
Is it currently easy to tell what version of the THEME was used?

### DISCUSSION THREAD

**Discussion by darrenaddy** [Lednerb/bilberry-hugo-theme#351] [Ideas]
Congrats on the 3,0,0 release/milestone! I'm really liking what I'm seeing so far.
I was thinking that it might be difficult to answer questions for theme users without knowing which specific version of the theme they are using. (Or perhaps there IS, and I just don't know how.) It would be best if one could View Source in the browser to see that, like you can with the Hugo version used to generate the page (in the "generator" meta tag).
`<meta name="generator" content="Hugo 0.92.1" />`

My thought was that it would be easy to insert into the comment viewable at the top of theme.js - because that can be called directly at https://siteurl/theme.js
One would just have to remember to keep the version number in the comment and the release version number in sync. Perhaps there is a better way? Or maybe just a dumb idea? :)

One gotcha to this (maybe) is that some people are making Hugo components available as a second theme. (Some require being listed first and some last). This does not strike me as a Best Practice, but I have seen it and Hugo allows it, by turning the .Site.Param.Theme into an array.

Thanks to all Bilberry contributors! Great work.

**Comment by igor-baiborodine** [comment]
Recently I switched my website [kiroule.com](https://www.kiroule.com/) to using the Bilberry theme as a Hugo module. With this approach, the current theme version can be found in the [go.mod](https://github.com/igor-baiborodine/kiroule.com/blob/master/go.mod) file. Thus, the version number can be extracted and added to the footer.

To do so, I created the [update-version.sh](https://github.com/igor-baiborodine/kiroule.com/blob/master/version/update-version.sh) script to update the `creditsText` setting in the configuration file:
```bash
#!/usr/bin/env bash

version=$(grep -o 'v[0-9]\.[0-9][0-9]\.[0-9]' go.mod)
sed -i "s/creditsText = .*$/creditsText = \"Bilberry Hugo Theme $version\"/" config/_default/config.toml
```
Since I deploy my website on Netlify, I adjusted the deploy command in the [netlify.toml](https://github.com/igor-baiborodine/kiroule.com/blob/master/netlify.toml) file:

![image](https://user-images.githubusercontent.com/1027701/182469396-1a23d5b5-001d-4e41-90b1-015a7b307c21.png)

![image](https://user-images.githubusercontent.com/1027701/182469897-6dfc0720-0cd9-43ed-be4f-452eb5c98802.png)
